confluent_server/confluent/auth.py

```diff
@@ -165,6 +206,7 @@ def authorize(name, element, tenant=False, operation='create',
     # skipuserobj is a leftover from the now abandoned plan to use pam session
     # to do authorization and authentication.  Now confluent always does authorization
     # even if pam does authentication.
+    check_for_yaml()
```

Comment:
This approach could significantly slow down requests, as every individual api request must go through authorize.